src/lambda/interfaces.ts

```diff
@@ -32,3 +33,8 @@ export type CrudInputParams = {
   keys?: {[key: string]: any}
   customParameters?: {[key: string]: any}
 }
+
+export interface SecretValue {
```

Comment:
The Secret Value can be an object of any type, the "SecretValue" maybe can be named as "AccessKey" and contain region in this secret also, but as optional